blocks/identity-block/features/account-management/default.test.jsx

```diff
@@ -25,21 +21,22 @@ describe("Account management", () => {
 
 	it("renders header text", async () => {
 		render(<AccountManagementPresentational header="header" />);
-		await expect(screen.findByText("header")).not.toBeNull();
+		await expect(screen.findByText("header")).not.toBeNull(); // eslint-disable-line
```

Comment:
This test is only passing because `screen.findByText("header")` returns a Promise, which is not null. The 'header' text is not actually in the output. I haven't thoroughly checked the other places where `// eslint-disable-line` was added, but they all look like the same issue. I'm having trouble finding when these tests were changed to be `findByText` instead of `getByText`, but when that change was made something broke with the way the component rendered in the test.

Reply:
@vgalatro even I have a question regarding to this. I didn't touch the files related to account-management files, However, when I started to check if the tests were passing, I started to see errors related with these files.  Not sure why it's happening, I think only the test files related with the changes on the branch should be considered.

Reply:
I'm not understanding. There are changes in this test file, the default.jsx, and two of the child component test files, so they would have been flagged by eslint. Are you saying you made none of those changes until eslint threw warnings/errors?

Reply:
Yes, It's correct

Reply:
Hi @vgalatro I started to use getByText again, but I'm facing linting errors. Any suggestion for this??

<img width="648" alt="Screenshot 2024-04-10 at 4 28 49 p m" src="https://github.com/WPMedia/arc-themes-blocks/assets/54566275/68b585a8-2c85-416d-a6a1-04f7dff75f90">

Reply:
@LauraPinilla This is valid lint warning. `findBy` methods are a combination of `getBy` and `waitFor`

Reply:
@malavikakoppula I was checking the documentation, but I couldn't find another option to solve this. The account-management component has a lot of `useState` and `useEffect` inside, I need to wait until all these changes are applied before I can try to check if the text exists. 
For example, on [L77](https://github.com/WPMedia/arc-themes-blocks/pull/2067/files#diff-971d48d87baac5b1417c572b57748cf2186178dd681c0b42dc9c3316b37e327cR77) It's easy since the text we are looking for is the title, and it's rendered immediately, but in this case, we need have async operations before rendering the text.